android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -163,7 +163,12 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
   protected @Nullable String mDownloadingMessage = null;
   protected @Nullable String mLackPermissionToDownloadMessage = null;
 
+  private static Set<String> originWhitelist;
+
   public RNCWebViewManager() {
+    originWhitelist = new HashSet<>();
```

Comment:
The variable name should likely mention `camera`, as we don't want this to be changed in the future to mix granular allowlists

Otherwise in future PRs the check below might degrade by adding more permissions to the websites that were added only for camera, and vice versa.

Reply:
renamed it to `cameraPermissionOriginWhitelist`